src/lm.jl

```diff
@@ -27,11 +27,12 @@ mutable struct LmResp{V<:FPVector} <: ModResp  # response in a linear model
         new{V}(mu, off, wts, y)
     end
 end
-LmResp(y::V) where {V<:FPVector} = LmResp{V}(fill!(similar(y), 0), similar(y, 0), similar(y, 0), y)
 
-LmResp(y::AbstractVector{T}) where T<:Real = LmResp(float(y))
+LmResp(y::FPVector; wts::FPVector = similar(y, 0)) = LmResp{typeof(y)}(fill!(similar(y), 0), similar(y, 0), wts, y)
```

Comment:
Make `wts` a positional argument for consistency with `GlmResp`.

Also, no spaces around `=` for arguments, and break lines at 92 chars.

Reply:
I didn't want to make `wts` a positional argument because there are 4 fields in `LmResp` and `wts` is like the 4th argument. So it seemed weird to have the arguments taken be different from the fields of the object.

Reply:
Maybe but then `GlmResp` should be changed too... Not sure it's worth it.